x-pack/test_serverless/functional/test_suites/common/examples/discover_customization_examples/customizations.ts

```diff
@@ -12,18 +12,18 @@ const TEST_START_TIME = 'Sep 19, 2015 @ 06:31:44.000';
 const TEST_END_TIME = 'Sep 23, 2015 @ 18:31:44.000';
 
 export default ({ getService, getPageObjects }: FtrProviderContext) => {
-  const PageObjects = getPageObjects(['common', 'timePicker', 'header']);
+  const PageObjects = getPageObjects(['common', 'timePicker', 'header', 'svlCommonPage']);
   const esArchiver = getService('esArchiver');
   const kibanaServer = getService('kibanaServer');
   const testSubjects = getService('testSubjects');
   const browser = getService('browser');
   const dataGrid = getService('dataGrid');
   const defaultSettings = { defaultIndex: 'logstash-*' };
 
-  // Flaky in serverless tests (before hook)
-  // Failing: See https://github.com/elastic/kibana/issues/165396
-  describe.skip('Customizations', () => {
+  describe('Customizations', () => {
     before(async () => {
+      // TODO: Serverless tests require login first
```

Comment:
```suggestion
      // Serverless tests require login first
```

Reply:
Leaving this comment here, but it applies to all the PRs with similar comments. I left the `TODO`s in any spots I had to change between the stateful and Serverless versions of the tests so that they are easy to identify and address when we do the work to merge these tests together later once deployment-agnostic tests are supported. So they're just acting as markers for now, and as part of the followup work we'll remove all `TODO`s from our Serverless tests 👍